helmfile/values/graphprotocol-indexer-agent.yaml

```diff
@@ -1,7 +1,12 @@
 postgres:
-  db: graph-node
-  host: postgres-postgresql
-  user: postgres
+  db: indexer
+  host: postgres-postgres
+  user: indexer
+  password:
+    fromSecret:
+      # Get password from secret generated by postgres operator
+      name: indexer.postgres-postgres.credentials.postgresql.acid.zalan.do
```

Comment:
Is this automatically generated?

Reply:
If yes, then I understand the reasoning behind the previous decisions regarding the postgresql password in other charts.